packages/jest-diff/README.md

```diff
@@ -292,6 +293,32 @@ const options = {
 
 A patch mark like `@@ -12,7 +12,9 @@` accounts for omitted common lines.
 
+### Example of option to include change counts
+
+To display the number of change lines at the right of annotation lines:
+
+```js
+const a = ['change from', 'common'];
+const b = ['change to', 'insert', 'common'];
+const options = {
+  includeChangeCounts: true,
+};
+
+const difference = diffLinesUnified(a, b, options);
+```
+
+```diff
+- Expected  1
```

Comment:
Have you considered the format
```
-1  Expected
+2  Received
```
? It is perhaps less likely to be mistaken for the total line counts of Expected/Received if the number is close together with the +/-.
Maybe there's prior art in other diff renderers?

Reply:
> What is your interpretation if you see a ratio?

Not very clear to me, esp. since the sign is still far from the number.

Brainstorming things to show both number (not final):
```
5 Expected (-1)
6 Received (+2)
```

But I'm not sure if showing the totals is a good idea at all, if you just look at 5,6 it looks like just one line differs when it really could be 11.